src/backend/catalog/aclchk.c

```diff
@@ -298,6 +298,14 @@ ExecuteGrantStmt(GrantStmt *stmt)
 	bool		added_objs = false;
 
 	/*
+	 * Don't allow GRANT/REVOKE if in ranger mode.
+	 */
+	if (aclType == HAWQ_ACL_RANGER)
+	{
+		elog(ERROR, "GRANT/REVOKE is not allowed in ranger mode");
+	}
+
+	/*
```

Comment:
I don't think we should forbid all the objects for grant/revoke. Given that we decide to leave catalog table/owner check in HAWQ side, we should allow grant/revoke for these operations.  What we need to forbid is what we have supported in Ranger UI configuration.